### DISCUSSION TITLE
Add a vertical header control to the dashboard

### DISCUSSION THREAD

**Discussion by cccs-jc** [apache/superset#18460] [Ideas]
**Is your feature request related to a problem? Please describe.**
Be able to put headers on the left/right vertically. It helps saving space on really crammed dashboards.

**Describe the solution you'd like**
A new control in the dashboard UI to create vertical headers.

**Describe alternatives you've considered**
We have a work around which involves putting html and css code in the markdown slice. However this solution is not very intuitive to use.

**Additional context**
This is what a dashboard with horizontal header and a custom markdown displaying vertical text. But a better solution would be to have a native vertical header.

![image](https://user-images.githubusercontent.com/56140112/111330071-04b90880-8646-11eb-88a3-90e734e3c792.png)

**Comment by rosanzheng** [comment]
I just started using Apache Superset and was exactly looking for this option. Is there an update to that feature?
